### PR TITLE
Adding isWatermarked to BoxItem.

### PR DESCRIPTION
**Pull request by dmaynard**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @dmaynard has signed the CLA. Thanks for the pull request!

File: src/main/java/com/box/sdk/BoxItem.java [hunk 1]
```diff
@@ -373,6 +376,15 @@ public String getItemStatus() {
             return this.collections;
         }
 
+
+        /**
+         * Gets flag indicating whether this file is Watermarked.
+         * @return whether the file is watermarked or not
+         */
+        public Boolean getIsWatermarked() {
```
Comment (gcurtis) [inline src/main/java/com/box/sdk/BoxItem.java]: This should be lowercase `boolean`.

File: src/main/java/com/box/sdk/BoxItem.java [hunk 2]
```diff
@@ -153,6 +153,9 @@ public abstract BoxSharedLink createSharedLink(BoxSharedLink.Access access, Date
         private BoxFolder.Info parent;
         private String itemStatus;
         private Set<BoxCollection.Info> collections;
+        private Boolean isWatermarked;
```
Comment (gcurtis) [inline src/main/java/com/box/sdk/BoxItem.java]: Lowercase `boolean` to use the primitive datatype.

File: src/main/java/com/box/sdk/BoxFile.java [hunk 3]
```diff
@@ -892,6 +893,14 @@ public URL getPreviewLink() {
             return this.previewLink;
         }
 
+        /**
+         * Gets flag indicating whether this file is Watermarked.
+         * @return whether the file is watermarked or not
+         */
+        public Boolean getIsWatermarked() {
```
Comment (gcurtis) [inline src/main/java/com/box/sdk/BoxFile.java]: This also needs to be lowercase `boolean`.

File: src/main/java/com/box/sdk/BoxFolder.java [hunk 4]
```diff
@@ -784,6 +785,14 @@ public boolean getCanNonOwnersInvite() {
             return this.canNonOwnersInvite;
         }
 
+        /**
+         * Gets flag indicating whether this file is Watermarked.
+         * @return whether the file is watermarked or not
+         */
+        public Boolean getIsWatermarked() {
```
Comment (gcurtis) [inline src/main/java/com/box/sdk/BoxFolder.java]: This also needs to be lowercase `boolean`.

**Comment by dmaynard** [comment]
Changed isWatermarked from Boolean to boolean

**Pull request merged**